x/ibc/02-client/client/utils/utils.go

```diff
@@ -65,24 +67,34 @@ func QueryClientStateABCI(
 		return nil, err
 	}
 
-	clientStateRes := types.NewQueryClientStateResponse(clientID, anyClientState, proofBz, res.Height)
+	// TODO: retrieve epoch-number from chain-id
+	height := types.NewHeight(0, uint64(res.Height))
+	clientStateRes := types.NewQueryClientStateResponse(clientID, anyClientState, proofBz, height)
 	return clientStateRes, nil
 }
 
 // QueryConsensusState returns a consensus state.
 // If prove is true, it performs an ABCI store query in order to retrieve the merkle proof. Otherwise,
 // it uses the gRPC query client.
 func QueryConsensusState(
-	clientCtx client.Context, clientID string, height uint64, prove, latestHeight bool,
+	clientCtx client.Context, clientID string, heightI exported.Height, prove, latestHeight bool,
 ) (*types.QueryConsensusStateResponse, error) {
 	if prove {
-		return QueryConsensusStateABCI(clientCtx, clientID, height)
+		return QueryConsensusStateABCI(clientCtx, clientID, heightI)
+	}
+	height, ok := heightI.(types.Height)
+	if !ok {
+		return nil, sdkerrors.Wrapf(
+			sdkerrors.ErrInvalidHeight, "Invalid height type: %T, expected: %T",
```

Comment:
```suggestion
			sdkerrors.ErrInvalidHeight, "invalid height type: %T, expected: %T",
```
golang recommends not having error messages start with capitals